public/index.html

```diff
@@ -2,12 +2,22 @@
 <html>
  <head>
    <title>TentHop Proxy</title>
+   <link rel="stylesheet" href="styling.css">
  </head>
  <body>
-  <div id="photo-module"></div>
-  <div id="photo-carousel-modal"></div>
+   <header>
+    <a id="logo"> <img src="https://tenthop.s3.us-east-2.amazonaws.com/tenthop+logo.png" alt="TentHop logo" width="125" height="125"/></a>
+  </header>
+  <div id="photo-carousel">
```

Comment:
To position the components, you should wrap all of the different modules into containers. From there, think about what containers should hold which component. 

- overall-container
  -top 
    -photos
  -bottom
    -left
      -info
  
...etc. 

Hopefully that makes sense and helps you!